process_scheduling_algorithms/preemptive_priority_scheduling.c

```diff
@@ -0,0 +1,112 @@
+#include <stdio.h>
+
+// Process structure containing ID, arrival time, burst time, priority, executed time, and completion time
+struct Process {
```

Comment:
For structure documentation refer to the contribution guidelines
Please document every function and structure as noted in the contribution guideline.

It should be a multiline comment in the following format

```
/**
 * @brief <the use of the code breifly>
 * @param <parameter name> <documentation of the parameter>
 * @return <if function what it returns>
 */
```
 For a more in-depth look try looking into doxygen documentation format because that's the documentation engine we use

Reply:
@realstealthninja  Thank you for your review I have made the changes suggested by you.